### DISCUSSION TITLE
Self-hosted on AWS ElasticBeanstalk

### DISCUSSION THREAD

**Discussion by robikovacs** [calcom/docker#164] [General] [answered]
Hi guys,

I'm trying to self-host cal.com on AWS with EB, I've created this `Dockerrun.aws.json` and deployed.

```json
{
  "AWSEBDockerrunVersion": "1",
  "Image": {
    "Name": "calendso/calendso:latest"
  },
  "Ports": [
    {
      "ContainerPort": "3000"
    }
  ]
}
```

It looks like the migrations run successfully
```sh
@calcom/prisma:db-deploy: 105 migrations found in prisma/migrations
@calcom/prisma:db-deploy: 
@calcom/prisma:db-deploy: 
@calcom/prisma:db-deploy: No pending migrations to apply.
...
@calcom/web:start: cache bypass, force executing 53c54b494709be00
@calcom/web:start: $ next start
@calcom/web:start: ready - started server on 0.0.0.0:3000, url: http://localhost:3000
```

But when I go to the website I just get an HTTP 408. Any ideas what can it be? The logs are not showing any errors.

**Comment by robikovacs** [comment, marked as answer]
I found the issue, it was a LB issue, I was using https instead of http for instance protocol.
<img width="750" alt="Screenshot 2022-09-15 at 14 04 55" src="https://user-images.githubusercontent.com/5718030/190388380-027c4469-514d-4cfe-abbd-b74ff73cc70a.png">

**Comment by krumware** [comment]
Thanks for the info!
